content/www/locales/fr.json

```diff
@@ -2,116 +2,117 @@
                 }
             }
         },
         "editorFeatures": {
-            "header": "The Ideoxan Lesson Editor",
-            "subheader": "A sneak peek at our robust, in-browser Integrated Development Environment",
+            "header": "L'éditeur de cours Ideoxan",
+            "subheader": "Un aperçu de notre environnement de développement intégré complet dans le navigateur",
             "items": {
                 "feature0": {
-                    "title": "Easy to Use",
-                    "description": "The editor uses a flat and minimal interface that anyone, from professional software developers to starters, can use. The editor resembles an actual Integrated Development Environment (IDE) meaning making the switch to an external program is easy."
+                    "title": "Simple à utiliser",
+                    "description": "L'editeur utilise une interface minimale que n'importe qui, débutant ou confirmé, peut utiliser. L'éditeur ressemble à une vraie IDE, il est donc plus simple de passer à un autre programme."
                 },
                 "feature1": {
-                    "title": "Responsive",
-                    "description": "The Editor works on multiple displays. This means that programming on any device is a breeze. Learning programming on the go has never been easier!"
+                    "title": "Adaptif",
+                    "description": "L'éditeur supporte toutes les tailles d'écran. Vous pouvez donc programmer depuis n'importe quel appareil, y compris en déplacement."
                 },
                 "feature2": {
-                    "title": "Real Time Editing",
-                    "description": "The viewport updates automatically to run your code in real time. Every time you make a change, your code is uploaded and saved so you can resume your work later."
+                    "title": "Edition en temps réel",
+                    "description": "L'apercu se met à jour automatiquement et exécute votre code en temps réel. Dès que vous faites un changement, votre code est sauvegardé en ligne pour que vous puissiez reprendre plus tard."
                 },
                 "feature3": {
-                    "title": "Document Manipulation",
-                    "description": "This is a step above your ordinary word processor or text editor. The online editor features syntax highlighting that is customizable, the ability to undo/redo, a tabbed document interface, keyboard shortcuts, error reporting, and much more."
+                    "title": "Manipulation de documents",
+                    "description": "Au-delà de votre éditeur de texte classique. Notre éditeur inclut une coloration syntaxique personnalisable, une fonction annuler/refaire, une interface en onglets, des raccourcis clavier, le signalement d'erreurs et beaucoup plus."
+			// Seriously?
```

Comment:
Please remove any and all comments since they break the translate utility.

Reply:
ok